web/src/main/webapp/admin_hunter_controllers.js

```diff
@@ -35,4 +35,46 @@ portalControllers.controller('AdminHuntersCtrl',
             );
         };
 
+    });
+
+function findHunterById($hunterId, $scope, $http) {
+    $http.get('/pa165/rest/hunter/' + $hunterId).then(function (response) {
+        $scope.hunter = response.data;
+        console.log('deleted hunter ' + hunter.id + ' (' + hunter.email + ')');
```

Comment:
please remove or change this log,
it is throwing  angular.js:12477 ReferenceError: hunter is not defined
and it is about find, not delete